cmd/api/main.go

```diff
@@ -119,6 +122,27 @@ func main() {
 		log.Fatalf("Failed to connect to database: %v", err)
 	}
 
+	sqlDB, err := db.DB()
+
+	// Set DB connection limits
+	maxIdle := serverConfig.DB_MAX_IDLE_CONNECTIONS
+	maxOpen := serverConfig.DB_MAX_OPEN_CONNECTIONS
+	if maxOpen < 1 {
```

Comment:
There's one more problem with this code is that elides standard library behavior. There's no way to get standard library behavior.

Reply:
Not sure I follow you point entirely.  Some clarification could help.

My best guess is that you don't want to have a default setting at all.

So that if they do not set the env vars, just do nothing, and leave the current behavior.

I'm fine with that change, if that is what you mean.

Reply:
Yes, that's what I meant. Or setting 0 means we are going to use the current behavior.